client/src/pages/Item.js

```diff
@@ -23,10 +24,30 @@ function Item({ state, dispatch, init }) {
 
 
```

Comment:
@MarkAlperin I'm sure you saw my earlier comment but just to be through, please confirm that both the Home.js and Item.js are matching the dev branch files. (And then add your changes of course lol) thanks @MarkAlperin

Reply:
double checked, looks good